datarobot_batch_scoring/batch_scoring.py

```diff
@@ -134,7 +137,7 @@ def run_batch_predictions(base_url, base_headers, user, pwd,
                     [c for c in keep_cols if c not in first_row.fieldnames],
                     first_row.fieldnames))
 
-        if not dry_run:
+        if not (dry_run or import_id):
```

Comment:
Actually if we don't fail here, workers will never abort, and just continue to push and fail each batch.

Name of this thing is really misleading, as it's not authorize, it's more like validate_prediction_params